packages/formation/js/back-to-top.js

```diff
@@ -58,7 +58,6 @@ export default function setup() {
   // The current page likely does not contain a "Back to top" button in its layout.
 
   const footer = document.getElementById('footerNav');
-  if (!footer) return;
```

Comment:
I'm a bit confused about how these changes solve the problem with the button not appearing in IE11. I see how not returning early will add the `scroll` handler, which will result in `breakpointCheck()` being called:


https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/c74674449c4a74018e5bf239e5069edc2fe75737/packages/formation/js/back-to-top.js#L39-L43